<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Counselor</class>
 <widget class="QWidget" name="Counselor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>758</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Counselor</string>
  </property>
  <widget class="QLabel" name="counselor_lbl_background">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-50</y>
     <width>771</width>
     <height>571</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background:#1A1627;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="counselor_txt_allAppointment">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>421</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="patient_combo_profession">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>60</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <item>
    <property name="text">
     <string>Dahiliye</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nöroloji</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Diş</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="patient_combo_doctor">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>100</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QDateEdit" name="patient_date_day">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="counselor_btn_delete">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>340</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="counselor_btn_addAppointment">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>180</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI Semilight</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Add Appointment</string>
   </property>
  </widget>
  <widget class="QLabel" name="counselor_txt_name">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:White;</string>
   </property>
   <property name="text">
    <string>Ad</string>
   </property>
  </widget>
  <widget class="Line" name="line_mid_a">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>280</y>
     <width>511</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="counselor_lbl_Image">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>30</y>
     <width>151</width>
     <height>151</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>151</width>
     <height>141</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid;
border-color:white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="counselor_txt_ID">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>41</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:White;</string>
   </property>
   <property name="text">
    <string>ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="counselor_txt_doctor">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:White;</string>
   </property>
   <property name="text">
    <string>Doctor : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="counselor_line_deleteID">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>300</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="counselor_txt_day">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>140</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:White;</string>
   </property>
   <property name="text">
    <string>Day : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QTextBrowser" name="counselor_txt_output">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>70</y>
     <width>191</width>
     <height>431</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="counselor_line_confirmID">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="patient_btn_delete_2">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Confirm</string>
   </property>
  </widget>
  <widget class="QLabel" name="patient_lbl_txt_info">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>40</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:White;</string>
   </property>
   <property name="text">
    <string>INFO</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="patient_line_ID_3">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>ID :</string>
   </property>
  </widget>
  <widget class="QLabel" name="counselor_txt_profession">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:White;</string>
   </property>
   <property name="text">
    <string>Profession : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="counselor_txt_patientID">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:White;</string>
   </property>
   <property name="text">
    <string>Patient ID : </string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="Line" name="line_mid_b">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>511</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_mid_c">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>20</width>
     <height>261</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_mid_d">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>20</width>
     <height>261</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
